reviewrot/__init__.py

```diff
@@ -297,7 +297,9 @@ def parse_cli_args(args):
                         metavar="CHANNEL",
                         default=None,
                         help='send output to list of irc channels')
-
+    parser.add_argument('--ignore-wip',
```

Comment:
Do you need a `default=False` here?

Reply:
I think 'action='store_true'' automatically sets the default to false. See here: https://stackoverflow.com/questions/8203622/argparse-store-false-if-unspecified

Reply:
Perfect!